viewsb/descriptor.py

```diff
@@ -126,8 +124,15 @@ def get_decoded_descriptor(self, data=None, use_pretty_names=True):
         if data is None:
             data = self.get_raw_data()
 
-        return self.decode_data_as_descriptor(data, use_pretty_names)
-
+        try:
+            return self.decode_data_as_descriptor(data, use_pretty_names)
+        except ConstError as error:
+            errorParts = str(error).split('\n')
+            what = error.path.split('->')[1].strip() if error.path else "error"
+            self.parsed = {what: errorParts[1]}
+        except:
```

Comment:
Same here.

Reply:
Noted. I'll create a fix for this over-catch tomorrow.